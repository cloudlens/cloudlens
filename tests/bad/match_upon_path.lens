match(entry) upon (e.bar.foo){
    "(?<bar>\d+)"
}
