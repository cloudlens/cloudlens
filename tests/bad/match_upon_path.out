Invalid path: e.bar.foo. Maybe you forgot the entry name.
